BiweeklyMeetings/2022-06-28.md

```diff
@@ -0,0 +1,442 @@
+**Guests**: Morgan Thomas, Michael Liesenfelt
+
+### Triage
+
+#### PR271
+[PR271 CIP-0027? | Update Proposal: Multiple Royalty Addresses](https://github.com/cardano-foundation/CIPs/pull/271)
+
+**Matthias** -: CIP-0027? | Update Proposal: Multiple Royalty Addresses
+  And that current CIP is already active. It's been used by a few marketplaces. So it doesn't quite make sense to modify it as much. As for the proposal itself, I made already some remark last week. I think the author came back saying yes, true. But that would make for a worse UX. And therefore I would prefer the approach. They have a client in the proposal, which is fair enough, but I still maintain that it should be made as a separate proposal, which is an extension of the existing one rather than modifying what exists and has been used in a while.
+
+**M. Àngels** -:  So I see you maintain a conversation. So from my understanding, Nicolas will need to make a separate, right?
+
+**Matthias** -: Yes, indeed.
+
+**M. Àngels** -:  So shall we add that we are waiting for the author?
+
+**Matthias** -: I think that's pretty much the case now because the proposal by itself...
+
+**Robert** -: [Issues with the audio] I don't think based on what Matthias was saying before, which he'd probably be staying now, if he could. It sounds like that can only really go ahead as a separate CIP because of the existing implementations that it would be incompatible with.
+
+**M. Àngels** -:  And apologies on my lack of knowledge on this. So in this case, what 
+is the next step.
+
+**Robert** -: As far as I know, we would say that this is waiting for author. And then 
+if that is the agreement, then we would deprecate this one in favor of the new PR once the author submits it.
+
+**M. Àngels** -:  Okay.
+
+**Matthias** -: I think it can also be the same PR right, it can just modify it.
+
+**Robert** -: I can add that label now waiting for author, if that's okay with everybody else.
+
+**Matthias** -: Yeah, good.
+
+**M. Àngels** -:  I was doing it already, so don't worry about this.
+
+**Matthias** -: So the content itself, I mean, I already admitted my remark last week and I agree with the author. I think I've just asked him to put that as a rational because I would avoid having this discussion again. Should be fine, but just as a separate content.
+
+**M. Àngels** -:  Perfect.
+
+**Matthias** -: Two different.
+
+### Last Check 
+
+#### PR263
+[PR263 CIP-0054? | Cardano Smart NFTs](https://github.com/cardano-foundation/CIPs/pull/263)
+
+Okay. So we move to the items on last check. So we have PR 263, CIP 54, Cardano smart NFTs.
+
+**Matthias** -: Yes. So this one was on last check, I think because it was substantially discussed on the forum. And this is like the results of quite a few weeks of discussions. Had another go at it today, actually. So made a few small remarks. It's mainly on the editorial aspect. So just titles and sections to stick a bit more to [crosstalk 00:03:17].
```

Comment:
```suggestion
**Matthias** -: Yes. So this one was on last check, I think because it was substantially discussed on the forum. And this is like the results of quite a few weeks of discussions. Had another go at it today, actually. So made a few small remarks. It's mainly on the editorial aspect. So just titles and sections to stick a bit more to the template.
```